wordpress-mobile/gutenberg-mobile
# Rename iOS gutenberg demo app to GutenbergDemo

With the update to React 0.61.5 the iOS demo project is now fully using CocoaPods to build itself.

This creates a scenario where sometimes Xcode build system gets confused between the pod library Gutenberg (the Gutenberg Bridge podspec) and the binary of the demo project itself: gutenberg.

You will see Xcode sometimes complaining that we are importing Gutenberg to gutenberg, and after getting on this start you will probably will need to clean all build folders and restart the project.

So the solution I propose here is to rename the `gutenberg.xcodeproj` and `gutenberg.xcodeworkspace` to `GutenbergDemo` and avoid these conflicts going forward.

To test:
 - Check if projects build
 - Check that tests build and pass

PR submission checklist:
